addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2365,6 +2387,28 @@ var SnippetsMenu = Widget.extend({
                 return this.nodeType === 3 && this.textContent.match(/\S/);
             }).parent().addClass('o_default_snippet_text');
     },
+    _touchHandler(event) {
```

Comment:
Add the missing JS doc + rename the `event` argument to `ev`. Using `event` is a source of error as it is the name of a magic global variable on Chrome only.